ModelBundle/Repository/NodeRepository.php

```diff
@@ -189,6 +190,22 @@ public function findByParentIdAndSiteId($parentId, $siteId)
     }
 
     /**
+     * @param string $path
+     * @param string $siteId
+     *
+     * @throws \Doctrine\ODM\MongoDB\MongoDBException
+     *
+     * @return mixed
+     */
+    public function findByPathAndSiteId($path, $siteId)
```

Comment:
This method replace findByParentIdAndSiteId which returns a subtree too. Should I be keep the naming rule or rename as you suggest. If the return was a single node, i would named the method findOneByPathAndSiteId.